### PR TITLE
Llelievr/battery shield defaults

### LINKED ISSUE #12: Different values for battery resistor depending on board selection
The default values for battery resistors should change for each board.
Also the default pins (not sure if they do already)
https://github.com/SlimeVR/SlimeVR-Tracker-ESP/commit/3337d5a5392324d662398a5943636e160a102c38

### PR DESCRIPTION
**Pull request by loucass003**
Closes #12

@unlogisch04 you can check this file to see the defaults values on each boards. I wont ask you to review the code ;) but please double check those defaults for me 🙏

### CODE REVIEW

File: backend/src/firmware/boards-infos.json [hunk 1]
```diff
@@ -54,7 +98,18 @@
...
         ],
-        "board": "esp12e"
+        "board": "esp12e",
+        "defaults": {
+            "imuSDA": "4",
+            "imuSCL": "5",
+            "imuInts": ["14", "13"],
+            "led": "2",
+            "enableLed": true,
+            "ledInverted": false,
```
Comment (unlogisch04) [inline backend/src/firmware/boards-infos.json:108]: `//  #define LED_INVERTED false`
In <defines.h> this is not defined -> default from <globals.h>
is ledInverted true

File: backend/src/firmware/boards-infos.json [hunk 2]
```diff
@@ -96,6 +162,17 @@
...
-        "board": "esp32dev"
+        "board": "esp32dev",
+        "defaults": {
+            "imuSDA": "2",
+            "imuSCL": "0",
+            "imuInts": ["255", "255"],
+            "led": "255",
+            "enableLed": false,
+            "ledInverted": false,
+            "batteryPin": "A0",
```
Comment (unlogisch04) [inline backend/src/firmware/boards-infos.json:173]:
```suggestion
            "batteryPin": "255",
```
https://github.com/SlimeVR/SlimeVR-Tracker-ESP/blob/main/src/defines.h#L136

**Review by unlogisch04 — CHANGES_REQUESTED** [review]
I think it was good that i did look over it.

In the defines.h for some boards the LED is disabled by default, and falls back to the board. If the board defines a LED it will use the onboard led. But for that the ´#define LED_PIN´ should not be in the file.

**Comment by loucass003** [comment]
Closing that PR, Making a bigger one with some big rewrites

**Pull request closed without merging**